### PR TITLE
Update docstring of torch.nn.functional.normalize()

### LINKED ISSUE #99125: F.normalize() to support multiple dimensions
### Motivation: 
I frequently encounter use cases where a tensor needs to be normalized along multiple dimensions at once, but `F.normalize(x, dim)` only supports `dim` being int. 
### Proposal: 
It will be convienent to support multi-dimensional normalization.
Thank you !

### Alternatives

_No response_

### Additional context

_No response_

cc @svekars @carljparker @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

### PR DESCRIPTION
**Pull request by mega-optimus**
Fixes #99125

torch.nn.functional.normalize() already supports dim=tuple(int), but the docstring says int only.

### CODE REVIEW

**Review by albanD — APPROVED** [review]
That looks good thanks!

**Comment by albanD** [comment]
Could you follow the instructions above to accept the CLA?

**Comment by mega-optimus** [comment]
> Could you follow the instructions above to accept the CLA?
I filled out the form about 4 times, don't know what's wrong.
